pycket/pycket-lang/zoTransform.rkt

```diff
@@ -0,0 +1,581 @@
+#lang racket
+
+(require compiler/zo-parse setup/dirs
+         (only-in pycket/expand hash* global-config))
+
+(provide to-ast-wrapper primitive-table)
+
+(define DEBUG #f)
+(define pycketDir (path->string (current-directory))) ;; MUST BE RUN UNDER PYCKET DIR
+(define collectsDir (path->string (find-collects-dir)))
+(define moduleName 'beSetBy-main)
+
+;; FROM
+;; https://github.com/racket/compiler/blob/master/compiler-lib/compiler/decompile.rkt#L14
```

Comment:
You're copying code from `decompile.rkt`, which is unfortunate. Better would be to have that file provide something that we could use.